.github/workflows/docker/Dockerfile-dev-centos8.2

```diff
@@ -0,0 +1,33 @@
+FROM centos:8.2.2004
+
+LABEL maintainer="YiLin Li <YiLin.Li@linux.alibaba.com>"
+
+RUN dnf clean all && rm -r /var/cache/dnf && \
+    dnf --enablerepo=PowerTools install -y wget make libseccomp-devel \
+    gcc git openssl-devel glibc-static binutils-devel protobuf-c-devel rpm-build
+
+WORKDIR /root
+
+# install go
+RUN wget https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz && \
+    tar -zxvf go1.14.2.linux-amd64.tar.gz -C /usr/lib && \
+    rm -rf go1.14.2.linux-amd64.tar.gz
+
+# configure GOPATH and GOROOT
+ENV GOROOT       /usr/lib/go
+ENV GOPATH       /root/gopath
+ENV PATH         $PATH:$GOROOT/bin:$GOPATH/bin
+ENV GOPROXY      "https://mirrors.aliyun.com/goproxy,direct"
```

Comment:
Please remove this line. This base docker image is common to use for any developers. We can use multi-stage building to involve this line for CI/CD testing.